tutorials/terraform/data-transfer/data-transfer-yds-mch.tf

```diff
@@ -0,0 +1,145 @@
+# Infrastructure for the Yandex Cloud Data Streams, Managed Service for ClickHouse cluster and Data Transfer.
```

Comment:
Infrastructure for the Yandex Cloud Data Streams, Managed Service for ClickHouse cluster, and Data Transfer.